alloc/src/virtual_alloc.rs

```diff
@@ -183,8 +183,37 @@ pub mod os {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use crate::utils::*;
     use allocator_api2::alloc::Allocator;
 
+    #[test]
+    fn fuzz() {
+        use bolero::TypeGenerator;
+
+        #[cfg(miri)]
+        const MAX_SIZE: usize = 1_000_000;
```

Comment:
Why is miri different here? Too slow on bigger sizes?

Reply:
Miri had a memout if we used too large a size